### ISSUE TITLE
Feature request: Display Changelog in app

### ISSUE THREAD

**Issue by Poolitzer** [TeamNewPipe/NewPipe#3100]
I would love to have that implemented. Maybe even with links to the fitting blog posts.

Even better would be to be able to see what the new version brings me from the notification, and then I decide if I want to download it. If we want to link to the blogpost there, we would be required to publish it before the new version, but that would also be nice.

**Comment by B0pol** [comment]
Related #2124.

**Comment by Poolitzer** [comment]
Not sure I see how they are related?

**Comment by TobiGr** [comment]
I create a first prototype for viewing changelogs yesterday :D
However, the changes are stored in a file which is included in the app so viewing the changelog before updating is not possible. I guess only a few people translate these files during the release phase.

I guess @B0pol wants to note that we can get the changelogs directly from the fastlane directory and thus offer translations of them, too. Is this correct?

**Comment by TobiGr** [comment]
I also considered this. We can either link the post before releasing a new version or even better. get that url from our api

**Comment by B0pol** [comment]
> > If we want to link to the blogpost there, we would be required to publish it before the new version, but that would also be nice.
> 
> I also considered this. We can either link the post before releasing a new version or even better. get that url from our api

> I guess @B0pol wants to note that we can get the changelogs directly from the fastlane directory and thus offer translations of them, too. Is this correct?

Yeah, that's what I meant by related, include translation of changelogs. And indeed if you use API, we can gather data after update, therefore get some time for translations (with F-Droid delay + RC phase it should be good).

**Comment by Poolitzer** [comment]
I am not sure I can follow 100%. But it sounds like doubling the work, fastelane and our API. Cant we use only one?

**Comment by opusforlife2** [comment]
- I think the blog post should be linked in the update notification.
- The release notes should be bullet points, nothing else.
- There can be a link to the blog post in the release notes.
- There should be an option to turn off showing release notes automatically.
- The release notes should be viewable at any time by tapping an option in settings.

> I just created a first prototype for viewing changelogs yesterday :D

Any progress on that, @TobiGr?

**Comment by Stypox** [comment]
Related: #859 and #1316 (about app tours and FAQs)

**Comment by tEctutoRIt** [comment]
![8dd9aa51-5f89-43d3-a958-ff163ea84701.jpg](https://user-images.githubusercontent.com/77356742/122400179-1c7f9000-cf99-11eb-8229-fdfd5132c327.jpg)

I too think it be nice have a direct link to the change log in this about page in app.
